pkg/sdn/plugin/sdn-cni-plugin/openshift-sdn.go

```diff
@@ -0,0 +1,511 @@
+// +build linux
```

Comment:
sdn/plugin/sdn-cni-plugin/ is kind of redundant and you could probably get away with calling this directory just "cni".

also, maybe move it up a level?